### ISSUE TITLE
Please cut a new tag to latest docker image

### ISSUE THREAD

**Issue by jennym12911** [dnsviz/dnsviz#101]
Hello! Could you cut a new v0.9.4 tag to the dnsviz/dnsviz docker image for the latest image? Thanks.

**Comment by tkrizek** [comment]
Published [dnsviz/dnsviz:v0.9.4](https://hub.docker.com/layers/dnsviz/dnsviz/v0.9.4/images/sha256-5a4c5e8293d591010bd57fb2e79a803865607d72457e74335229130f69265932?context=repo).

Steps to build the image:
```
$ git checkout v0.9.4
$ git clean -dfx  # to make sure we have a clean build
$ podman build --network host -t docker.io/dnsviz/dnsviz:v0.9.4 -f Dockerfile
$ podman tag docker.io/dnsviz/dnsviz:v0.9.4 docker.io/dnsviz/dnsviz:latest  # tag as latest as well
$ podman login docker.io
$ podman push docker.io/dnsviz/dnsviz:v0.9.4
$ podman push docker.io/dnsviz/dnsviz:latest
```

Using `docker` instead of `podman` command works as well and you can the omit the qualifying `docker.io` part of the image name.

**Comment by cdeccio** [comment]
Thank you so much, @tkrizek !

@jennym12911 , hopefully this helps!
